pkg/controller/generate-registry/generate.go

```diff
@@ -316,35 +316,37 @@ func findCertificate(assetNames map[string]struct{}, checksumAssetName string) s
 	return ""
 }
 
-func checkChecksumCosign(pkgInfo *registry.PackageInfo, checksumAssetName string, assetNames map[string]struct{}) *registry.Cosign {
-	signatureAssetName := findSignature(assetNames, checksumAssetName)
-	if signatureAssetName == "" {
-		return nil
-	}
-	var pubKeyAssetName string
-	if !strings.HasSuffix(signatureAssetName, "-keyless.sig") {
-		// If the signature is keyless, a public key isn't used
-		pubKeyAssetName = findPubKey(assetNames)
+func findCosignBundle(assetNames map[string]struct{}, assetName string) string {
+	for _, suf := range []string{".cosign.bundle", ".bundle", ".sigstore", ".sigstore.json"} {
```

Comment:
I haven't seen all these for checksums files in the wild, but I don't see why they would not work, so included them all here.